### PR TITLE
Add pem utils for private keys, upgrade Play to 2.2.5

### PR DESCRIPTION
**Pull request by eric-wu**

### CODE REVIEW

File: app/org/sagebionetworks/bridge/crypto/PemUtils.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        StringReader stringReader = null;
+        try {
+            stringReader = new StringReader(text);
+            bufferedReader = new BufferedReader(stringReader);
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                lines.add(line);
+                line = bufferedReader.readLine();
+            }
+        } finally {
```
Comment (eric-wu) [inline app/org/sagebionetworks/bridge/crypto/PemUtils.java]: Java 7 can auto-close resources for you

File: app/org/sagebionetworks/bridge/crypto/PemUtils.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            if (begin.equals(line)) {
+                addLine = true;
+            }
+        }
+        Base64 encoder = new Base64(Base64.PEM_CHUNK_SIZE);
+        byte[] decoded = encoder.decode(base64Encoded.toString());
+        return decoded;
+    }
+
+    private static List<String> readLines(String text) throws IOException {
```
Comment (DwayneJengSage) [inline app/org/sagebionetworks/bridge/crypto/PemUtils.java]: Would it be cleaner to replace this with String.split()? This way, we can avoid having to create streams and readers. If we're worried about windows newlines vs unix newlines, we could just use a regex that looks like "\r?\n".

File: app/org/sagebionetworks/bridge/crypto/PemUtils.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        } catch (InvalidKeySpecException e) {
+            throw new RuntimeException(e);
+        } catch (NoSuchProviderException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static byte[] decode(List<String> lines, String begin, String end) {
+        checkNotNull(lines, "Lines cannot be null");
+        checkArgument(lines.size() >= 2, "Two few lines.");
```
Comment (DwayneJengSage) [inline app/org/sagebionetworks/bridge/crypto/PemUtils.java]: Nitpick: Typo - "too"

File: app/org/sagebionetworks/bridge/crypto/PemUtils.java (hunk 2 above)
Comment (eric-wu) [inline app/org/sagebionetworks/bridge/crypto/PemUtils.java]: You are right "\r?\n" is what BufferedReader does and is what I need here.  I can certainly replace it with String.split().

**Comment by DwayneJengSage** [comment]
Looks good to me.

**Pull request merged**